themes/bootstrap3/templates/checkouts/listhistory.phtml

```diff
@@ -36,11 +38,56 @@
         <?=$this->context($this)->renderInContext('myresearch/controls/sort.phtml', ['sortList' => $this->sortList]); ?>
       <?php endif; ?>
     </nav>
+    <?php if ($purgeAllAllowed || $purgeSelectedAllowed): ?>
+      <form name="purge_history" method="post" action="<?=$this->url('checkouts-purgehistory')?>" id="purge_history" data-disable-on-submit>
+      <input type="hidden" id="submitType" name="purgeSelected" value="1">
+      <input type="hidden" name="csrf" value="<?=$this->escapeHtmlAttr($this->auth()->getManager()->getCsrfHash())?>">
+      <div class="toolbar">
+        <div class="checkbox">
+          <?php if ($purgeSelectedAllowed): ?>
+            <label>
+              <input type="checkbox" name="selectAll" class="checkbox-select-all"/>
+              <?=$this->transEsc('select_page')?>
+            </label>
+            <div class="btn-group">
+              <input type="submit" id="purgeSelected" name="purgeSelected" value="<?=$this->transEscAttr('loan_history_purge_selected')?>" class="btn btn-default dropdown-toggle" data-toggle="dropdown" data-lightbox-ignore>
```

Comment:
We should keep an eye on #2723 -- we might need to refactor this to a component once that gets merged (and since there's some custom JS/CSS related to this control here, we should make sure any special use cases here are covered by the component).